### ISSUE TITLE
Typo on Options page (minor)

### ISSUE THREAD

**Issue by jtrickey** [Dan-in-CA/SIP#197]
Under "Station Handling", the tooltip for "Individual Duration" states "Allow each station to have its own rum time in programs."

On line 150 of gv.py "rum" should be "run".  The sprinklers aren't allowed to party without me.  :)

Software is great, and has really helped me out. Thank you so much!

**Comment by Dan-in-CA** [comment]
jtricky,

Thanks for that. This actually comes at the perfect time.

Some users have been interested in running SIP on other hardware (Pi clones) such as orange Pi and Odroid-C2. I have decided to set up separate branches for those and keep them synced by applying updates as patches rather than merging which would combine all the code.

This minor update will give me the opportunity to  try out the patch method.

I'll let you know when the update has been added.
